components/docs-chef-io/content/automate/security_best_practices.md

```diff
@@ -0,0 +1,22 @@
++++
+title = "Security Best Practices"
+
+draft = false
+
+gh_repo = "automate"
+[menu]
+  [menu.automate]
+    title = "Security Best Practices"
+    parent = "automate/reference"
+    identifier = "automate/reference/security_best_practices.md Security Best Practices"
+    weight = 50
++++
+
+## Insuring autocomplete remains disabled on the login screen.
+
+A configuration maintained in automate determines to autocomplete functionality in the nginx config.
```

Comment:
@vinay033 , @dishanktiwari2501  can we reword as- A configuration maintained in Chef Automate's `nginx.conf` file determines the autocomplete functionality.